packages/strapi-helper-plugin/lib/src/utils/inputsValidations.js

```diff
@@ -63,7 +66,15 @@ const validateInput = (value, inputValidations = {}, type = 'text') => {
     }
   });
 
-  if (type === 'email' && !emailRegex.test(value)) {
+  if (type === 'email') {
+    parseDomainResult = parseDomain(value);
+  }
+
+  if (
+    type === 'email' &&
+    !emailRegex.test(value) &&
+    !(parseDomainResult.type === ParseResultType.Listed)
+  ) {
```

Comment:
Can you gather the two ifs please ?
Something like:
```javascript
if (type === 'email') {
  parseDomainResult = parseDomain(value);

  if (!emailRegex.test(value) || parseDomainResult.type !== ParseResultType.Listed) {
    errors.push({ id: 'components.Input.error.validation.email' });
  }
}
```

Reply:
@petersg83 
     Yeah, I'll modify the same as nested if

Reply:
@petersg83 
Mentioned changes are committed.